version/distro/distro.go

```diff
@@ -75,6 +77,9 @@ func linuxDistro() Distro {
 	case have("/usr/local/bin/freenas-debug"):
 		// TrueNAS Scale runs on debian
 		return TrueNAS
+	case isUDMPro():
```

Comment:
Maybe move this to the bottom since it's more expensive than the `have()` tests?

Reply:
@sailorfrag Thank you for the suggestion. I understand the concern about performance, but this check MUST remain above `case have("/etc/debian_version")`. Since it runs on Debian, moving it down would result in it being incorrectly identified as Debian instead of UDMPro.

Reply:
I might of found a file that's unique to udm pro, but I have no idea if it's on 100% of UDM Pro's or only older/newer/etc...

```text
/usr/share/initramfs-tools/scripts/ui-boot-udm
```

Reply:
Oh right, I'm sorry, I remember going over this before, it had just been so long. Its current position is required so that makes sense to leave as it is. We can refine that later if needed.

Reply:
I found some other possibilities after searching install packages on a udm pro. Leaving this here for history so I (or someone else) can refine later in a follow up PR.

```text
# apt list --installed | egrep -i 'udm|pro'

dnscrypt-proxy/oldstable,now 2.0.45+ds1-1+b5 arm64 [installed]
e2fsprogs/oldstable-security,now 1.46.2-2+deb11u1 arm64 [installed]
improxy/now 1.1-30+g5a9d153d10e arm64 [installed,local]
iproute2/now 5.10.0-4-5257+gc8faeca5ad7 arm64 [installed,upgradable to: 6.1.0-3~bpo11+1]
libboost-program-options1.74.0/oldstable,now 1.74.0-9 arm64 [installed]
libprocps8/oldstable,now 2:3.3.17-5 arm64 [installed]
libprotobuf-c1/oldstable,now 1.3.3-1+b2 arm64 [installed]
libprotobuf23/oldstable,now 3.12.4-1+deb11u1 arm64 [installed]
procps/oldstable,now 2:3.3.17-5 arm64 [installed]
python3-protobuf/oldstable,now 3.12.4-1+deb11u1 arm64 [installed]
python3-unifi-console-protos/now 0.5.0-10+g908e06a184c7 all [installed,local]
udm-gd-lcm-fw/now 2.4.7-0-gf682009 all [installed,local]
udm-initramfs/now 2.0 all [installed,local]
udm-pro-lcm-fw/now 2.4.7-0-gf682009 all [installed,local]
udm-pro-rec/now 0.6.0-10+gb86d1d60117a arm64 [installed,local]
udmpro-al324-base-files/now 4.0.104~1193+g54d4a9b1 all [installed,local]
unifi-assets-udmpro/now 3.0.1 all [installed,local]
unifi-protect/bullseye,now 5.1.78 arm64 [installed]
wifiman-proxy/now 1.0.2-23+g94472150bf5 arm64 [installed,local]
```

```text
root@UDM-Pro:~# dpkg -L unifi-assets-udmpro
/.
/usr
/usr/share
/usr/share/unifi-assets
/usr/share/unifi-assets/images
/usr/share/unifi-assets/images/1024.png
/usr/share/unifi-assets/images/48.png
/usr/share/unifi-assets/images/64.png
/usr/share/unifi-assets/intro.mp4
/usr/share/unifi-assets/manifest.json
/usr/share/unifi-assets/setup
/usr/share/unifi-assets/setup/troubleshooting
/usr/share/unifi-assets/setup/troubleshooting/connected-eth.png
/usr/share/unifi-assets/setup/troubleshooting/connected-sfp.png
/usr/share/unifi-assets/setup/troubleshooting/no-internet-eth.png
/usr/share/unifi-assets/setup/troubleshooting/no-internet-sfp.png
/usr/share/unifi-assets/setup/troubleshooting/plug-cable.mp4
/usr/share/unifi-assets/setup/troubleshooting/restart-modem-eth.mp4
/usr/share/unifi-assets/setup/troubleshooting/restart-modem-sfp.mp4
/usr/share/unifi-assets/setup/troubleshooting/testing-connection-eth.mp4
/usr/share/unifi-assets/setup/troubleshooting/testing-connection-sfp.mp4
```

```text
root@UDM-Pro:~# dpkg -L udm-pro-lcm-fw
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/udm-pro-lcm-fw
/usr/share/doc/udm-pro-lcm-fw/changelog.gz
/usr/share/doc/udm-pro-lcm-fw/copyright
/usr/share/firmware
/usr/share/firmware/udm-pro-lcm-fw.bin
/usr/share/firmware/udm-pro-lcm-fw.hash
/usr/share/firmware/udm-pro-lcm-fw.version
```